From 0bbecde643f35e3f75847ff3a5cd2247a25baaf3 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Tue, 4 Jul 2023 12:05:44 +0200
Subject: [PATCH 0361/1695] dts: stm32: enable RTC on stm32mp257f-ev1

Set RTC status to "okay" in stm32mp257f-ev1.dts

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 1a674aeee2d2409d5f74c6372fdc69e897e490ab dts: stm32: enable RTC on stm32mp257f-ev1]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I2ab928cdfb79f952dce5e744c27cdb19f8047e6f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/316196
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/349407
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index c1e4d29f1..d5c73f4ad 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -54,6 +54,10 @@
 	status = "okay";
 };
 
+&rtc {
+	status = "okay";
+};
+
 &usart2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&usart2_pins_a>;
-- 
2.47.2

